src/open_inwoner/templates/pages/contactmoment/detail.html

```diff
@@ -1,21 +1,42 @@
 {% extends 'master.html' %}
-{% load i18n anchor_menu_tags card_tags file_tags grid_tags table_tags solo_tags form_tags button_tags %}
+{% load i18n grid_tags table_tags button_tags icon_tags %}
 
 {% block content %}
     {% if contactmoment %}
         {% render_grid %}
             {% render_column span=12 %}
-                {# Contactmoment/dashboard. #}
-                <h1 class="h1" id="title">{{ page_title }}</h1>
+                <h1 class="h1" id="title">{{ contactmoment.onderwerp }}</h1>
                 {% include "components/Dashboard/Dashboard.html" with metrics=metrics only %}
             {% endrender_column %}
-            {% render_column start=4 span=6 %}
+        {% endrender_grid %}
 
-                <div class="contactmoment__details">
-                {% contactmoment_table contactmoment zaak=zaak %}
+        <div class="container--no-margin">
+            <div class="contactmoment">
+                <h3>{% trans "Uw vraag" %}</h3>
+                <p>{{ contactmoment.text }}</p>
+            </div>
+            <div class="contactmoment">
+                <h3>{% trans "Antwoord" %}</h3>
+                {% if contactmoment.antwoord %}
+                    <p>{{ contactmoment.antwoord }}</p>
+                {% else %}
+                    <p>{% trans "Nog geen antwoord beschijkbaar" %}</p>
```

Comment:
All your paragraphs still need the proper line-height and `color: var(--font-color-body);` you can do this by either giving them the 'p class or create a contactmoment-p class where you can control their spacing as well.
(Our default body font color is not black: giving it a bit of a gray color is better for accessibility).